src/App.test.tsx

```diff
@@ -1,9 +1,33 @@
-import { render, screen } from "@testing-library/react"
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
we need a little more for our smoke test, for more check the commit message: https://github.com/sopra-fs22-group-36/screw-your-neighbor-react/pull/17/commits/0bf19e69860536e27f9ffa22c343d67eedc48dd5